ui/app/AppLayouts/Wallet/views/collectibles/CollectibleView.qml

```diff
@@ -49,12 +63,13 @@ Item {
             horizontalAlignment: Text.AlignLeft
             verticalAlignment: Text.AlignVCenter
             font.pixelSize: 15
-            color: Theme.palette.directColor1
+            customColor: Theme.palette.directColor1
             font.weight: Font.DemiBold
             elide: Text.ElideRight
-            text: isLoaded ? root.collectibleModel.name : "..."
+            text: isLoadingDelegate ? Constants.dummyText : d.isLoaded && d.modeDataValid ? root.collectibleModel.name : "..."
+            loading: root.isLoadingDelegate
         }
-        StatusBaseText {
+        StatusTextWithLoadingState {
```

Comment:
`Layout.preferredWidth: 88`, instead, to better fit the design?

Reply:
I am not sure what you mean?

Reply:
I mean that the title and subtitle loading rectangles doesn't have the same width than in design. 134 and 88 are the values there! Also the spacing between them is slightly less in the pr.

Maybe with a picture you will understand better! 

Here the design:
![Screenshot 2023-02-02 at 13 42 43](https://user-images.githubusercontent.com/97019400/216328307-512cdf35-20d1-44e3-b2a2-4022e78fc9b7.png)

Here the current pr:
![Screenshot 2023-02-02 at 13 44 31](https://user-images.githubusercontent.com/97019400/216328449-679547f6-1aba-4f61-a511-23fd7b3b57da.png)

Reply:
done . the spacing. between the 2 could be a design issue as I have I am reusing the same component everywhere.